### PR TITLE
 Add Document and fix some problems

### PR DESCRIPTION
**Pull request by Medicean**
- Add Docker document
- link mongodb and es container
- server starts after es(start slow)
- web & server depends_on mongodb & es
- remove web volumes (cause install.go os.Rename trigger Invalid cross-device link Error)
- add openssl in ids_web (install.go step 4 need)
- move all web files in /web directory (looks like clear)

### CODE REVIEW

File: Dockerfile-server [hunk 1]
```diff
@@ -19,4 +15,4 @@ RUN apk add --update tzdata
 ENV TZ=Asia/Shanghai
 RUN rm -rf /var/cache/apk/*
 RUN chmod +x server
-ENTRYPOINT ./server -db $mongodb -es $elasticsearch
+ENTRYPOINT for i in $(seq 1 5);do echo "Wait for ElasticSearch Start...$i"; curl -s http://$IDS_ELASTICSEARCH> /dev/null ; if [ $? = 0 ];then break; fi; sleep 1; if [ $i == 5 ];then echo "Too much retry, exit."; exit 1; fi; done; ./server -db $IDS_MONGODB -es $IDS_ELASTICSEARCH
```
Comment (deamwork) [inline Dockerfile-server]: 🤔I think to put all of these into a shell script and make that as the entry point would be clear.

**Review by deamwork — CHANGES_REQUESTED** [review]
Just need a little change. Others LGTM. 👍

File: Dockerfile-server (hunk 1 above)
Comment (Medicean) [reply Dockerfile-server]: okay

**Review by deamwork — APPROVED** [review]

**Review by deamwork — APPROVED** [review]
LGTM, will merge later. Thx. 🎉😄

**Pull request merged**